pkgs/development/haskell-modules/configuration-ghc-8.4.x.nix

```diff
@@ -40,6 +40,10 @@ self: super: {
   unix = null;
   xhtml = null;
 
+  # Restricts aeson to <1.4
+  # https://github.com/purescript/purescript/pull/3537
+  purescript = doJailbreak super.purescript;
```

Comment:
Will the jailbreak not fix the constraint on `base`, too? What happens when you try to build the jailbroken version with `ghc-8.6.3`? Does it not compile?

Reply:
IIRC it won't build due to `MonadFail` errors.